modules/core/include/opencv2/core/hal/intrin_sse.hpp

```diff
@@ -1329,10 +1329,9 @@ inline v_int8x16 v_absdiffs(const v_int8x16& a, const v_int8x16& b)
     v_int8x16 m = a < b;
     return (d ^ m) - m;
  }
-inline v_int16x8 v_absdiffs(const v_int16x8& a, const v_int16x8& b)
+static inline v_int16x8 v_absdiffs(const v_int16x8& a, const v_int16x8& b)
 { return v_max(a, b) - v_min(a, b); }
 
-
 inline v_int32x4 v_fma(const v_int32x4& a, const v_int32x4& b, const v_int32x4& c)
```

Comment:
Sorry. I forget revert changes in this file. I'll do it together with applying comments.

Reply:
Done.